src/dialogs/config-flow/show-dialog-options-flow.ts

```diff
@@ -52,10 +52,17 @@ export const showOptionsFlowDialog = (
       deleteFlow: deleteOptionsFlow,
 
       renderAbortDescription(hass, step) {
-        const description = hass.localize(
-          `component.${configEntry.domain}.options.abort.${step.reason}`,
-          step.description_placeholders
-        );
+        let description: string | undefined;
+        if (step.reason === "not_implemented") {
```

Comment:
We should not add specific logic for specific reasons, or we will have a long list of reasons here from different integrations. If the integration does not provided a translation, we should just use the reason as text.

Reply:
Broadly speaking, I agree; however, this message isn’t from the integration. It results from having no config flow available which is still an issue for many integrations.

Reply:
I thought about what you said more. Instead of showing the `not_implemented` message, we should actaully stop before even getting to that point. I will now instead stop us from even being able to click the button in that instance.

Reply:
Yes, we should never open config or option flows for integrations that dont support it. I still think we should just fallback to the `reason` here and not add more logic.